engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/schemaregistry/SchemaRegistryKafkaAvroProvider.scala

```diff
@@ -19,7 +17,7 @@ class SchemaRegistryKafkaAvroProvider[T](schemaRegistryProvider: SchemaRegistryP
     schemaRegistryProvider
```

Comment:
let's add some comment that currently we're doing approximate type generation to avoid false positives in validation, but this should be improved in the future

Reply:
Done